README.md

```diff
@@ -9,18 +9,24 @@ These files are in SMIRKS Native Open Force Field (SMIRNOFF) format.
 Details about this new format are documented in our recent publication ([doi:10.1021/acs.jctc.8b00640](https://www.doi.org/10.1021/acs.jctc.8b00640) or [bioRxiv](https://doi.org/10.1101/286542)), and the most recent documentation of the specification can be found in the [Open Force Field Toolkit documentation](https://open-forcefield-toolkit.readthedocs.io/en/latest/smirnoff.html).
 You can parameterize small molecules with SMIRNOFF using the
 `ForceField` class in the [openforcefield toolkit](https://github.com/openforcefield/openforcefield)
-for simulations with [OpenMM](http://openmm.org/).
+for simulations with [OpenMM](http://openmm.org/), or other simulation formats supported by [ParmEd](http://parmed.github.io/ParmEd/html/index.html).
 
 Usage examples can be found in the [openforcefield repository](https://github.com/openforcefield/openforcefield/tree/master/examples).
 
+Each force field is currently available in two forms -- Both with and without bond constraints to hydrogen. 
+The default version of each force field has bond constraints to hydrogens, and is suitable for long-timestep simulations.  
+The `unconstrained` version of each force field is suitable for operations such as molecule single point energy calculations and geometry optimizations, and should be used when comparing the force field to QM data. 
```

Comment:
I'd change the order here, because "for operations such as" doesn't give explain which operations before giving an example. Maybe "should be used when single-point energies are a major concern (e.g. single point energy calculations and geometry optimizations) and should be used when comparing the force field to QM data."

Reply:
Changed in 7812e48